### DISCUSSION TITLE
Pipelines - Broken or how do they work?

### DISCUSSION THREAD

**Discussion by HorstBaerbel** [vcync/modV#853] [Q&A]
Say I want to render some color bars and use them as an input for color-mapping multiple groups / modules. I thought this was what pipelines were for. I'd expect the output to be never visible, but still rendered / generated in the background if I enable the pipeline / group. But whether I check the pipeline box or not makes absolutely no difference... What am I doing wrong?
[EDIT]: I've found the Alt-Click "Skip drawing to output canvas" feature which seems to do the right thing. Shouldn't that rather be a proper checkbox in the group settings?! And them wth are pipelines for?!
![image](https://github.com/vcync/modV/assets/1310747/eef34da6-242b-4417-93c8-95b28f8f3614)

**Comment by 2xAA** [comment]
@HorstBaerbel Here's what pipeline does: https://modv.vcync.gl/v3/documentation/groups.html#pipeline

"Skip drawing to output canvas" does what you want it to do, based on your use-case.
It could be its own checkbox, but there's not much room left in the UI, group settings would start to become quite long.

I did see #854 which might be some sort of solution to this, however I'd prefer to redesign the group controls rather than just making them taller.

**Reply by HorstBaerbel** [reply]
I've read the docs, but still do not understand what pipeline can be used for. Isn't a group with a single module and "Skip drawing to output canvas" basically the same thing?
